README.md

```diff
@@ -1,4 +1,255 @@
+	```
+
+	Approve the certificate signing request generated by Kubernetes:
+
+	```
+	kubectl certificate approve default.client.root
+	```
+	
+<!-- - Non-Kubernetes CA:
+
+	If you didn't use `cockroach cert` to generate the client certificate, substitute an appropriate `secretName` before applying this file:
+
+	```
+	kubectl create -f https://raw.githubusercontent.com/cockroachdb/cockroach-operator/master/examples/client-secure-byoc.yaml
+	``` -->
+
+You can then access the SQL shell and run SQL statements from this pod:
+
+```
+kubectl exec -it cockroachdb-client-secure -- ./cockroach sql --certs-dir=/cockroach-certs --host=cockroachdb-public
+```
+
+If you deployed an insecure cluster, you can access the SQL shell directly:
+
+```
+kubectl run cockroachdb -it --image=cockroachdb/cockroach:v20.1.4 --rm --restart=Never -- sql --insecure --host=cockroachdb
+```
+
+If you want to [access the Admin UI](#access-the-admin-ui), create a SQL user with a password while you're here:
+
+```
+CREATE USER roach WITH PASSWORD 'Q7gc8rEdS';
+```
+
+```
+\q
+```
+
+## Access the Admin UI
+
+To access the cluster's [Admin UI](https://www.cockroachlabs.com/docs/v20.1/admin-ui-overview), port-forward from your local machine to one of the pods:
+
+```
+kubectl port-forward cockroachdb-0 8080
+```
+
+Access the Admin UI at `https://localhost:8080`. Note that [certain Admin UI pages](https://www.cockroachlabs.com/docs/v20.1/admin-ui-overview#admin-ui-access) require `admin` access.
+
+## Configure the cluster
+
+You can configure the cluster by modifying and reapplying `example.yaml`.
+
+<!-- ### Expand disk size
+
+To expand persistent volumes, edit `resources.requests.storage`:
+
+```
+resources:
+  requests:
+    storage: 100Gi
+```
+
+Apply the new configuration:
+
+```
+kubectl apply -f example.yaml
+``` -->
+
+### Scale the cluster
+
+To scale the cluster, edit `nodes`:
+
+```
+nodes: 4
+```
+
+Do **not** scale down to fewer than 3 nodes. This is considered an anti-pattern and will cause errors.
+
+> Note that you must scale by updating the `nodes` value in the Operator configuration. Using `kubectl scale statefulset <cluster-name> --replicas=4` will result in new pods immediately being terminated.
+
+Apply the new configuration:
+
+```
+kubectl apply -f example.yaml
+```
+
+### Ugprade the cluster
+
+Perform a rolling upgrade by changing the image name:
+
+```
+image:
+  name: cockroachdb/cockroach:v20.1.4
+```
+
+Apply the new configuration:
+
+```
+kubectl apply -f example.yaml
+```
+
+## Stop the cluster
+
+Remove the Operator:
+
+```
+./hack/delete-operator.sh -c <cluster-name>
+```
+
+Clean up any resources associated with the `cockroachdb` label:
+
+```
+kubectl delete pods,statefulsets,services,persistentvolumeclaims,persistentvolumes,poddisruptionbudget,jobs,rolebinding,clusterrolebinding,role,clusterrole,serviceaccount -l app=cockroachdb
```

Comment:
Please do not tell this to do this.  If we have stuff leftover we need to address that as an issue.

Reply:
OK, I will verify and file a bug if any resources are being left over.